_source/_posts/2022-07-12-hosted-auth-cli.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: blog_post
+title: "Get Started with the Okta CLI and Hosted Authentication in Python"
+author: erikka-innes
+by: contractor
+communities: [python]
+description: "Learn how to set up the Okta CLI and run the hosted authentication sample application."
+tags: []
+tweets:
+- "Configure the Okta CLI and run your first sample application using hosted authentication!"
+- "Try the Okta CLI, the easiest way to create your first Okta application with hosted authentication."
+- ""
+image:
+type: conversion
+---
+
+Okta is a customizable, secure solution that lets you add authentication and access management to your applications. Since this can be a complex topic, we recommend setting up your first project with our CLI. In this walk through, we will:
```

Comment:
```suggestion
Okta is a customizable, secure solution that lets you add authentication and access management to your applications. Since this can be a complex topic, we recommend setting up your first project with Okta's CLI. In this walkthrough, we will:
```
treated in dev docs as one word